flow/metrics.yaml

```diff
@@ -69,3 +81,44 @@ components:
               x-field-uid: 1
             cut_through:
               x-field-uid: 2
+    Flow.TxRxReplication:
+      description: |-
+        Tx Rx replication factor is required to determine the ratio of expected number of Rx packets across all Rx ports
+        to the Tx packets for the configured flow.
+      type: object
+      properties:
+        choice:
+          type: string
+          default: one
```

Comment:
`one` makes much more sense for `TxRxRatio` than for `TxRxReplication`

Reply:
Yes, have updated to TxRxRatio.